app/scripts/reducers/config.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+import _config from '../config';
+
+export default (state = { ..._config }) => state;
```

Comment:
Add our configuration values into the `store` for easy access in components.

Reply:
Philosophically consistent with the redux method of `keeping all ui state in the store`